### PR TITLE
323 optional chemo fields

### PR DESCRIPTION
**Pull request by hknahal**
Related to https://github.com/icgc-argo/argo-dictionary/issues/323

### CODE REVIEW

**Comment by hknahal** [comment]
Tested in QA using dictionary version `114.4`

**Comment by hknahal** [comment]
@kcullion I was considering renaming the following fields so they are more clear. Let me know what you think:

Rename `form_dose_intensity_reduction` to `dose_intensity_reducing_event`
Update description to say "If there was a signficiant reduction in dose intensity, indicate which event caused it."

Rename `percentage_dose_intensity_reduction` to `amount_dose_intensity_reduction`
Update description to say "If there was a significant reduction in dose intensity, indicate the amount."

**Comment by kcullion** [comment]
@hknahal as discussed on slack...you could call the three fields
`dose_intensity_change` `dose_intensity_change_type` and `dose_intensity_change_amount`

**Comment by hknahal** [comment]
@kcullion I updated the fields as discussed (updated to `dose_intensity_reduction`, `dose_intensity_reduction_event`, and `dose_intensity_reduction_amount`). You can test the above example submission files (found [here](https://github.com/icgc-argo/argo-dictionary/pull/336#issuecomment-956411357)) on QA (dictionary version `114.6`)

**Comment by hknahal** [comment]
@kcullion I updated the fields as discussed (updated to `dose_intensity_reduction`, `dose_intensity_reduction_event`, and `dose_intensity_reduction_amount`). You can test the above example submission files (found [here](https://github.com/icgc-argo/argo-dictionary/pull/336#issuecomment-956411357)) on QA (dictionary version `114.6`)

File: schemas/chemotherapy.json [hunk 1]
```diff
@@ -104,6 +104,50 @@
...
+      "valueType": "string",
+      "restrictions": {
+        "codeList": "#/list/yes_no"
+      },
+      "meta": {
+        "displayName": "Dose Intensity Reduction"
+      }
+    },
+    {
+      "name": "dose_intensity_reduction_event",
```
Comment (kcullion) [inline schemas/chemotherapy.json:120]: @hknahal is this the part of the code that comes through on the docs website?
If so, just wondering if you describe in the Notes/Scripts that `dose_intensity_reduction_event` and `dose_intensity_reduction_amount` should be submitted if `dose_intensity_reduction` is Yes?

Comment (hknahal) [reply schemas/chemotherapy.json:120]: Thanks @kcullion Good point - I should a note to indicate this.

Comment (hknahal) [reply schemas/chemotherapy.json:120]: @kcullion I added a "notes" section to both these fields to indicate this.

**Review by kcullion — APPROVED** [review]
Looks great @hknahal !

**Pull request merged**